tools/amp_video/contact_sheets_collection.xml

```diff
@@ -0,0 +1,28 @@
+<tool id="collection_contact_sheets" name="Collection Contact Sheets" version="1.0.0">
+  <description>Contact Sheet Creation</description>
+  <requirements>
+	<requirement type="package" version="3.8">python</requirement>
+    <requirement type="package" version="8.30">coreutils</requirement>
+    <requirement type="package" version="3.4">ffmpeg</requirement>
+  </requirements>
+  <command detect_errors="exit_code">
+  	'$__tool_directory__/contact_sheets_collection.py' '$input_video' '$context_json' '$contact_sheet'
+  </command>
+  <inputs>
+	<param name="input_video" type="data" format="video" label="Input Video" help="An input video file"/>
+    <param name="context_json" type="text" label="HMGM Task Context" help="JSON string containing context information needed for creating a MGM task" >
```

Comment:
How about "JSON string containing context information needed by the MGM", since we are not creating any task here, Can you update this in applause_detection_to_avalon_xml.xml as well?